### ISSUE TITLE
GitHub link on top right corner is broken

### ISSUE THREAD

**Issue by lauthieb** [lauthieb/code-notes#150]
**Describe the bug**

When I click on the GitHub link in top right corner, it doesn't work and I get the following error message in my console "can't read property "shell" of undefined. (Thanks Kody for reporting me this bug).

*To Reproduce**
1. Open the developers tools
2. Click on the GitHub button on top right corner
3. You will see the error message

![image](https://user-images.githubusercontent.com/9600228/88237518-afc41e80-cc7f-11ea-940a-8bf6e5ec0829.png)

**Expected behavior**
It should redirect on https://github.com/lauthieb/code-notes.

**Comment by wchen928** [comment]
Hi @lauthieb, Is there anyone else working on this issue? May I help? I think there's also the same issue when you click: About > "Feel free to contribute on Github 🍻"

Thank you for this project btw! It's great.

**Comment by wchen928** [comment]
I fixed it without using the electron shell command. It worked, but I'm not sure if you had other preferred ways to fix it. 

This is what I put inside the methods 

```
methods: {

    openGithub: function(link) {

      window.open(link, "_blank");
    }
```
And this is how I put the link:
```
      <a
        id="github"
        @click="openGithub('https://github.com/lauthieb/code-notes')"
        title="Github"
      >
        <b-icon icon="github"></b-icon>
      </a>
```

Let me know if this helps. Thanks

**Comment by lauthieb** [comment]
Thanks! Can you make a Pull Request to fix this?

**Comment by wchen928** [comment]
#153 I just submit a pull request. Thanks!

**Comment by lauthieb** [comment]
Closed by #153
